### PR TITLE
Set release date for 0.8.1.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-### 0.8.1 (unreleased)
+### 0.8.1 (2020-12-17)
```
Comment (bshastry) [inline Changelog.md]: Why is the release back dated?

Comment (chriseth) [reply Changelog.md]: That is a very good question!

Comment (chriseth) [reply Changelog.md]: Fixed.

**Review by bshastry — APPROVED** [review]
LGTM

**Pull request merged**